install/zowe-install.yaml.template

```diff
@@ -27,4 +27,10 @@ terminals:
   sshPort=22
   telnetPort=23
 
+# started task JCL member for Zowe job - these entries are not case-sensitive
+zowe-server-proclib:
+# dsname=SYS1.PROCLIB   # Example of required dataset name.  Do not enclose it in quotes.
+  dsname=auto
```

Comment:
Make these field names camelCase to match the others

Reply:
Done.